BUILD

```diff
@@ -54,3 +55,48 @@ go_binary(
     embed = [":go_default_library"],
     visibility = ["//visibility:public"],
 )
+
+
+go_binary(
+    name = "bazelisk-darwin",
+    out = "bazelisk-darwin_amd64",
+    embed = [":go_default_library"],
+    goarch = "amd64",
```

Comment:
I didn't know that one could create platform-specific go_binary targets - that's much better than my existing approach in `build.sh`! :) Thanks!

Reply:
Yeah I figured you might want to refactor that script too. If nothing else, this allows you to build all the architectures in parallel, even if you still loop through them to place them in a `bin` directory